index.js

```diff
@@ -1,17 +1,46 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (!this.items[pos]) throw new Error('OutOfBounds')
 
-  max() {}
+    return this.items[pos]
+  }
 
-  min() {}
+  max() {
+    if (this.length === 0) throw new Error('EmptySortedList')
+    const pos = this.length - 1;
+    return this.items[pos]
+  }
 
-  sum() {}
+  min() {
+    if (this.length === 0) throw new Error('EmptySortedList')
+    return this.items[0]
+  }
 
-  avg() {}
+  sum() {
+    if (this.length === 0) return this.length;
+
+    const sum = this.items.reduce((acc, item) => {
```

Comment:
Boa! Aqui daria pra ter feito o reduce em uma linha somente
Se vc passar o ZERO como segundo argumento do reduce, vc nem precisa fazer o IF acima ;-)

return this.items.reduce((sum, num) => sum + num, 0);